specifyweb/frontend/js_src/lib/components/Statistics/index.tsx

```diff
@@ -328,9 +387,23 @@ export function StatsPage(): JSX.Element | null {
     [handleChange]
   );
 
+  const refreshPage = () => {
```

Comment:
replace with `function refreshPage...` syntax - use const for single expression functions only. use function for function with a statement, more than one expression, or if it's a hook/react component